lib/hanami/commands/server.rb

```diff
@@ -17,6 +17,9 @@ module Commands
     # @since 0.1.0
     # @api private
     class Server < ::Rack::Server
+      ENTR_EXECUTE_COMMAND = 'find . | entr -r bundle exec hanami server %s'.freeze
+      ENTR_CHECK_COMMAND = 'which entr'.freeze
```

Comment:
@aderyabin Using `which` makes this code non portable on Windows. Please check if this snippet can work for us: http://stackoverflow.com/a/5471032/498386